SPECS/mozjs/mozjs.spec

```diff
@@ -0,0 +1,112 @@
+%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+
+%global	major 78
+Summary:       Mozilla's JavaScript engine.
+Name:          mozjs
+Version:       78.10.0
+Release:       2%{?dist}
+Group:         Applications/System
+Vendor:        Microsoft Corporation
+License:       MPLv2.0 and MPLv1.1 and BSD and GPLv2+ and GPLv3+ and LGPLv2+ and AFL and ASL 2.0
+URL:           https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey
+Source0:       https://ftp.mozilla.org/pub/firefox/releases/%{version}esr/source/firefox-%{version}esr.source.tar.xz
+%define sha1   firefox-%{version}=547bac33732774993bd8223de9acfeedcc69a10d
+Patch0:        emitter.patch
+Patch1:        emitter_test.patch
+# Build fixes
+Patch2:        init_patch.patch
+Patch3:        spidermonkey_checks_disable.patch
+Distribution:  Mariner
+BuildRequires: which
+BuildRequires: python3-xml
+BuildRequires: python3-libs
+BuildRequires: python3-devel
+BuildRequires: zlib-devel
+BuildRequires: gcc
+BuildRequires: llvm
+BuildRequires: llvm-devel
+BuildRequires: icu-devel
+BuildRequires: rust
+BuildRequires: autoconf213
+Requires:      icu
+Requires:      python3
+Requires:      python3-libs
+Obsoletes:     mozjs60
+Obsoletes:     mozjs68
```

Comment:
Can we add a `Provides: mozjs%{major}` line here, to be in line with Fedora's naming style?

Reply:
done